<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>8-Bit Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_Intel8080">
       <property name="text">
        <string>Intel8080</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Tutorial">
       <property name="text">
        <string>Tutorial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
